pkg/controller/postgresql.go

```diff
@@ -352,6 +357,13 @@ func (c *Controller) warnOnDeprecatedPostgreSQLSpecParameters(spec *acidv1.Postg
 		(spec.EnableReplicaLoadBalancer != nil || spec.EnableMasterLoadBalancer != nil) {
 		c.logger.Warnf("Both old and new load balancer parameters are present in the manifest, ignoring old ones")
 	}
+
+	memoryRequest, err1 := strconv.Atoi(spec.ResourceRequests.Memory)
+	memoryLimit, err2 := strconv.Atoi(spec.ResourceLimits.Memory)
```

Comment:
are you sure this will work ? 
both limits are request have the following format: 
> ResourceDescription{CPU: "10m", Memory: "50Mi"},

Reply:
fix it.